.circleci/config.yml

```diff
@@ -13,6 +13,66 @@ postgres_service: &postgres_service
     POSTGRES_USER: postgres
     POSTGRES_DB: circle_database
 
+cache_1: &cache_1
+  key: npm-cache-01-{{ .Environment.CIRCLE_JOB }}-{{ checksum "/tmp/checksums.txt" }}-1
+  paths:
+    - ./node_modules
+    - ./yarn.lock
+    - packages/opentelemetry-base/node_modules
+    - packages/opentelemetry-scope-base/node_modules
+    - packages/opentelemetry-types/node_modules
+    - packages/opentelemetry-scope-async-hooks/node_modules
+    - packages/opentelemetry-core/node_modules
+    - packages/opentelemetry-exporter-prometheus/node_modules
+    - packages/opentelemetry-metrics/node_modules
+    - packages/opentelemetry-tracing/node_modules
+    - packages/opentelemetry-exporter-jaeger/node_modules
+    - packages/opentelemetry-exporter-zipkin/node_modules
+    - packages/opentelemetry-node/node_modules
+    - packages/opentelemetry-shim-opentracing/node_modules
+    - packages/opentelemetry-web/node_modules
+    - packages/opentelemetry-plugin-dns/node_modules
+
+cache_2: &cache_2
+  key: npm-cache-02-{{ .Environment.CIRCLE_JOB }}-{{ checksum "/tmp/checksums.txt" }}-1
+  paths:
+    - packages/opentelemetry-plugin-grpc/node_modules
```

Comment:
Is it possible to use `packages/opentelemetry-*/node_modules`?

Reply:
Why do you use `cache_1` and `cache_2`, is it possible to combine them together?

Reply:
No, paths don't support globbing, which is why the current caching is broken. The multiple caches have to be there because you can only cache so many paths in a single cache. Something to do with a limitation in the circle backend.

Reply:
I see, how many paths are allowed in one cache? Also, Whenever we add a new package, we should add it here also (`packages/opentelemetry-xx/node_modules` and `packages/opentelemetry-xx/yarn.lock`) and in case if we missed it the build will still pass right?

Reply:
I'm not sure what the actual limitation is. It seems like it's not actually a hard limitation on the number of paths, but some generated value overflows in the circle ci backend causing builds to crash. The error message is something about a header length.

Yes we will need to add new packages here, and the build will still pass if you miss it. It will also still pass if you add a path that doesn't exist.